src/avram/migrator/alter_table_statement.cr

```diff
@@ -127,6 +127,25 @@ class Avram::Migrator::AlterTableStatement
     {% end %}
   end
 
+  macro add_polymorphic_belongs_to(type_declaration, foreign_key_type = Int64, optional = false)
+    {{ nilable = !optional }}
+    rows << Avram::Migrator::Columns::{{ foreign_key_type }}Column.new(
+      name: {{ "#{type_declaration.id}_id" }},
+      nilable: !{{ nilable }},
+      default: nil
+    )
+    .build_add_statement_for_alter
+
+    rows << Avram::Migrator::Columns::StringColumn.new(
+      name: {{ "#{type_declaration.id}_type" }},
+      nilable: !{{ nilable }},
+      default: nil
+    )
+    .build_add_statement_for_alter
```

Comment:
I believe you can simplify these to `add {{ type_declaration.id}_type : StringColumn{% if nilable %}?{% end %}`

That way you don't need to manually use the column builder classes

Reply:
This way, as this is not `nilable` it's asking for a `default_value`, but for me makes no sense to ask for a default value for an association heheh

the error message

> Try one of these...
> 
>   ▸ add optional_commentable_id : Int64, default: "Something"
>   ▸ add optional_commentable_id : Int64, fill_existing_with: "Something"
>   ▸ add optional_commentable_id : Int64, fill_existing_with: :nothing
> 
>         add_polymorphic_belongs_to :optional_commentable, optional: true

```crystal
add {{ type_declaration.id }}_id : {{ foreign_key_type }}{% if nilable %}?{% end %}
add {{ type_declaration.id }}_type : String{% if nilable %}?{% end %}
```

any idea?

Reply:
I would pass `fill_existing_with: :nothing`